themes/odin-theme/static/css/style.css

```diff
@@ -88,13 +110,20 @@ body {
 footer {
   margin: 1em 0 0;
   padding: 1em 0 0;
-  text-align: center;
-  color: #ccc;
+  color: rgb(173, 173, 173);
```

Comment:
Why is rgb used here?

Reply:
VS Code has a built in color selector that uses rgb.